cli/script/command-executor.ts

```diff
@@ -1174,6 +1253,66 @@ function release(command: cli.IReleaseCommand): Promise<void> {
         });
 }
 
+export var releaseReact = (command: cli.IReleaseReactCommand): Promise<void> => {
+    var platform: string = command.platform.toLowerCase();
+    var entryFile: string = command.entryFile;
+    var outputFolder: string = path.join(os.tmpdir(), "CodePush");
+    var releaseCommand: cli.IReleaseCommand = <any>command;
```

Comment:
This <any> cast feels a little fragile because common members between the two interfaces (IReleaseCommand and IReleaseReactCommand) are expected to stay in sync.  Perhaps a common base interface or a comment (at the interface definition sites) would help.